### PR TITLE
feat: openapi changes for eject_expense_from_report API

### PR DESCRIPTION
**Pull request by sourav014**

### CODE REVIEW

File: src/admin/paths/admin@reports@eject_expenses.yaml [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+    content:
+      application/json:
+        schema:
+          type: object
+          additionalProperties: False
+          required:
+            - data
+          properties:
+            data:
+              $ref: '../../components/schemas/report.yaml#/report_action_with_expenses_ids_to_eject'
```
Comment (kartikeyrajvaidya) [inline src/admin/paths/admin@reports@eject_expenses.yaml:18]: where is this written?

Comment (kartikeyrajvaidya) [reply src/admin/paths/admin@reports@eject_expenses.yaml:18]: not able to find the definition of `report_action_with_expenses_ids_to_eject`

Comment (kartikeyrajvaidya) [reply src/admin/paths/admin@reports@eject_expenses.yaml:18]: Please check

Comment (sourav014) [reply src/admin/paths/admin@reports@eject_expenses.yaml:18]: report_action_with_expenses_ids_to_eject:
  type: object
  additionalProperties: False
  required:
    - id
    - expense_ids
  properties:
    id:
      type: string
      description: |
        id of report, from which the expenses are to be ejected
      example: 'rpa21A2rdj2'
    expense_ids:
      type: array
      items:
        allOf:
          - $ref: './fields.yaml#/id_string'
      example: [ 'txelntgwgwll' ]

Comment (sourav014) [reply src/admin/paths/admin@reports@eject_expenses.yaml:18]: https://github.com/fylein/fyle-platform-docs/blob/main/src/components/schemas/report.yaml

Screenshot 2023-05-23 at 10 25 35 AM

**Review by kartikeyrajvaidya — APPROVED** [review]

**Pull request merged**